insonmnia/dwh/server.go

```diff
@@ -1328,6 +1328,9 @@ func (w *DWH) onDealChangeRequestUpdated(eventTS uint64, changeRequestID *big.In
 }
 
 func (w *DWH) onBilled(eventTS uint64, dealID, payedAmount *big.Int) error {
+	w.mu.Lock()
```

Comment:
This function doesn't fit in two screens(which is a problem itself), and is being processed under lock. BTW I don't get why do you even need it.

Reply:
what does function size have to do with this PR? processing this event requires exactly N things to happen, and those things do not occur anywhere else in the code; I can’t see any way to reduce this code, if you have any ideas — please inform me